.github/workflows/build_and_test.yml

```diff
@@ -5643,7 +5643,7 @@ jobs:
       if: steps.cache-artifact.outputs.cache-hit != 'true'
```

Comment:
In this case I think we're OK since the actual contents of the OpenSSL and Xerces-C++ aren't changing (or maybe are just changing implementation but not API).  

The older vcpkg commit was seemingly trying to download sources from URLs which no longer worked, so when it intended to rebuild it failed.
https://github.com/objectcomputing/OpenDDS/runs/5838982830?check_suite_focus=true#step:7:1